src/components/alarmList/alarmList.js

```diff
@@ -133,7 +133,8 @@ class AlarmList extends Component {
         error={this.state.error}
         title={lang.ALARMSTATUS}>
         <div className="grid-container">
-          <AlarmsGrid {...alarmsGridProps} />
+          {this.state.rowData &&
+          <AlarmsGrid {...alarmsGridProps} pagination = {this.state.rowData.length > 8 ? true: false}/>}
```

Comment:
is "8" the right value in all the situations? if I remember correctly the number of alarms in a page depends on the browser window size, the zoom settings etc.